src/include/tcop/tcop.h

```diff
@@ -87,24 +121,32 @@ class TrafficCop {
   int BindParameters(std::vector<std::pair<int, std::string>> &parameters,
                      Statement **stmt, std::string &error_message);
 
+  void SetTcopTxnState(concurrency::Transaction * txn) {
+    tcop_txn_state_.emplace(txn, ResultType::SUCCESS);
+  }
+
+  ResultType CommitQueryHelper();
+
+  bool prepared_ = false;
```

Comment:
Please make it private. Also, why should we have this variable? Please give comments here.